blocks/careers/careers.js

```diff
@@ -0,0 +1,119 @@
+import { FACEBOOK, TWITTER, PINTEREST, LINKEDIN } from '../../scripts/constants.js';
+
+async function loadJobsData(sortBy, filterBy, block) {
+  try {
+    const response = await fetch(`https://careers.jobscore.com/jobs/pricefx/feed.json?sort=${filterBy}${sortBy}`, {
```

Comment:
@SwathiPrasadRaju Can we move this URL to url-constants.js ?

Reply:
Can you use this `ffetch` global method to get the response? Please refer `header.js`

Reply:
Since the Global function needs some code tweaks to handle the object response. So we can go with the `fetch` method.

Reply:
Thanks @Vaitheesaran-nagalingam  for confirming

Reply:
@amoghdaryapurkar  done with the changes....